src/odemis/driver/test/smaract_test.py

```diff
@@ -756,7 +756,9 @@ class TestMCS2_1DOF(unittest.TestCase):
     """
     @classmethod
     def setUpClass(cls):
-        cls.dev = smaract.MCS2(**CONFIG_1DOF)
+        # cls.dev = smaract.MCS2(**CONFIG_1DOF)
+        cls.microscope = model.getMicroscope()
+        cls.dev = model.getComponent(role="focus")
```

Comment:
I don't think you can merge this as-is. I understand this is to test while the backend is running. But that's not how standard test cases are run.